src/player.cpp

```diff
@@ -7756,6 +7756,18 @@ bool player::wield( item& target )
     if( target.is_null() ) {
         return true;
     }
+
+    // Query whether to draw an item from a holster when attempting to wield the holster
+    if( target.get_use( "holster" ) && ( target.contents.size() > 0 ) )
+    {
+        if( query_yn( string_format( _( "Draw %s from %s?" ),
+                                    target.get_contained().tname().c_str(),
```

Comment:
... and we need one more space and on line below.

If you feel it takes to much efforts to get minor improve then simply ignore it, it will not be a blocker, at least for me.